docs/sources/send-data/k8s-monitoring-helm/_index.md

```diff
@@ -0,0 +1,304 @@
+
+<!-- INTERACTIVE page step4.md END -->
+
+<!-- INTERACTIVE page step5.md START -->
+
+## Step 7: Accessing Grafana
+
+To access Grafana, you will need to port-forward the Grafana service to your local machine. To do this, run the following command:
+
+```bash
+export POD_NAME=$(kubectl get pods --namespace meta -l "app.kubernetes.io/name=grafana,app.kubernetes.io/instance=grafana" -o jsonpath="{.items[0].metadata.name}") && \
+kubectl --namespace meta port-forward $POD_NAME 3000 --address 0.0.0.0
+```
+
+{{< admonition type="tip" >}}
+This will make your terminal unusable until you stop the port-forwarding process. To stop the process, press `Ctrl + C`.
+{{< /admonition >}}
+
+This command will port-forward the Grafana service to your local machine on port `3000`. You can access Grafana by navigating to [http://localhost:3000](http://localhost:3000) in your browser. The default credentials are `admin` and `adminadminadmin`.  One of the first places you should visit is Explore Logs which will provide a no-code view of the logs being stored in Loki:
+
+[http://localhost:3000/a/grafana-lokiexplore-app](http://localhost:3000/a/grafana-lokiexplore-app)
+
+{{< figure max-width="100%" src="/media/docs/loki/k8s-logs-explore-logs.png" caption="Explore Logs view of K8s logs" alt="Explore Logs view of K8s logs" >}}
+
+## Step 8 (Optional): View the Alloy UI
+
+The Kubernetes Monitoring Helm chart deploys Grafana Alloy to collect and forward telemetry data from the Kubernetes cluster. The Helm is designed to abstract you from creating an Alloy configuration file. However if you would like to understand the pipeline you can view the Alloy UI. To access the Alloy UI, you will need to port-forward the Alloy service to your local machine. To do this, run the following command:
+
+```bash
+export POD_NAME=$(kubectl get pods --namespace meta -l "app.kubernetes.io/name=alloy-logs,app.kubernetes.io/instance=k8s" -o jsonpath="{.items[0].metadata.name}") && \
+kubectl --namespace meta port-forward $POD_NAME 12345 --address 0.0.0.0
+```
+
+{{< admonition type="tip" >}}
+This will make your terminal unusable until you stop the port-forwarding process. To stop the process, press `Ctrl + C`.
+{{< /admonition >}}
+
+This command will port-forward the Alloy service to your local machine on port `12345`. You can access the Alloy UI by navigating to [http://localhost:12345](http://localhost:12345) in your browser.
+
+{{< figure max-width="100%" src="/media/docs/loki/k8s-logs-alloy-ui.png" caption="Grafana Alloy UI" alt="Grafana Alloy UI" >}}
+<!-- INTERACTIVE page step5.md END -->
+
+<!-- INTERACTIVE page step6.md START -->
+
+## Step 9: Adding a sample application to `prod`
+
+Finally, lets deploy a sample application to the `prod` namespace that will generate some logs. To deploy the sample application run the following command:
+
+```bash
+helm install tempo grafana/tempo-distributed -n prod
```

Comment:
Received error message after running this command
"Error: INSTALLATION FAILED: cannot re-use a name that is still in use"

Reply:
Have you already run this step before? As helm won't alloy you to deploy the same name twice 
<img width="715" alt="Screenshot 2025-02-04 at 09 54 31" src="https://github.com/user-attachments/assets/7a88439a-afd4-4bb4-8fbe-4202aae230a6" />